daemon/routes/iprule/iprule.go

```diff
@@ -16,11 +17,12 @@ import (
 
 // Router uses `ip rule` under the hood
 type Router struct {
-	rpFilterManager routes.RPFilterManager
-	ifgroupManager  ifgroup.Manager
-	tableID         uint
-	fwmark          uint32
-	mu              sync.Mutex
+	rpFilterManager  routes.RPFilterManager
+	ifgroupManager   ifgroup.Manager
+	tableID          uint
+	fwmark           uint32
+	prevAllowSubnets []string
```

Comment:
[minor] regarding the naming, shouldn't it be `allowSubnets`, because it contains the list with the current allow list?

Reply:
probably you are right, just wanted member variable to be different from parameter name for better readability, but `prev` does not fit